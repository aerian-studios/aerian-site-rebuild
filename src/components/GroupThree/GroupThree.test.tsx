import * as React from "react";

import * as renderer from "react-test-renderer";

import { GroupThree } from "./index";

describe("GroupThree", () => {
    it("renders correctly", () => {
        const tree = renderer
            .create(
                <GroupThree className="myClass">
                    <div style={{ backgroundColor: "red", height: 200 }} />
                    <div style={{ backgroundColor: "green", height: 200 }} />
                    <div style={{ backgroundColor: "blue", height: 200 }} />
                </GroupThree>
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
