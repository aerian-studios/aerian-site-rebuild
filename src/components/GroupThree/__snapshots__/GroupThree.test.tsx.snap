// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GroupThree renders correctly 1`] = `
<div
  className=" myClass"
>
  <div
    style={
      Object {
        "backgroundColor": "red",
        "height": 200,
      }
    }
  />
  <div
    style={
      Object {
        "backgroundColor": "green",
        "height": 200,
      }
    }
  />
  <div
    style={
      Object {
        "backgroundColor": "blue",
        "height": 200,
      }
    }
  />
</div>
`;
